common/src/main/resources/application.common.properties

```diff
@@ -2,5 +2,7 @@ efs.mount=${AB2D_EFS_MOUNT:#{systemProperties['java.io.tmpdir']+'/jobdownloads'}
 
 audit.files.ttl.hours=24
 
-hpms.base.url=${AB2D_HPMS_URL:#{'https://hpmsimpl.cms.gov'}}
-hpms.auth.url=${AB2D_HPMS_AUTH_URL:#{'https://hpmsimpl.cms.gov/api/idm/oauth/token?ACS=wlVuThEThipRlBu37Pra'}}
\ No newline at end of file
+hpms.base.url=${AB2D_HPMS_URL:https://hpmsimpl.cms.gov}
+# The following parameter translates into a map to make it easy to add any number of parameters.
+# Example syntax: {firstLabel:'Some Value 1',secondLabel:'Some Value 2'}
+hpms.api.params=${AB2D_HPMS_API_PARAMS:{ACS:'wlVuThEThipRlBu37Pra'}}
```

Comment:
Git is complaining about no new line

Reply:
That's a weird complaint as the file historically did not have a new line at the end.